ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -107,7 +108,7 @@ export const itwEidIssuanceMachine = setup({
             target: "WalletInstanceAttestationObtainment"
           },
           {
-            target: "UserIdentification"
+            target: "IpzsPrivacyAcceptance"
```

Comment:
I think we should move the wallet instance creation after the user accepted the Privacy Policy. Currently the wallet instance is created right after accepting the TOS, before the privacy policy. @LazyAfternoons what do you think?